functions/pipeline/onboarding/__init__.py

```diff
@@ -9,10 +26,106 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
         req_body = req.get_json()
         logging.error(req.get_json())
         url_list = req_body["imageUrls"]
-        url_string = (", ".join(url_list))
     except ValueError:
         print("Unable to decode JSON body")
         return func.HttpResponse("Unable to decode POST body", status_code=400)
 
     logging.error(req_body)
-    return func.HttpResponse("Got body: " + url_string, status_code=200)
\ No newline at end of file
+
+    # Build list of image objects to pass to DAL for insertion into DB.
+    image_object_list = []
+    image_name_list = []
+    
+    # TODO: Add check to ensure image URLs sent by client are all unique.
+
+    for url in url_list:

```

Comment:
@ackaufman this loop should be encapsulated in a method. Moreover we may want that method to be wrapped in a try/catch and send an appropriate http response. E.G. if the issue is our code then send http code 400

Reply:
TODO note added